projects/ui/src/lib/components/po-chart/samples/sample-po-chart-labs/sample-po-chart-labs.component.html

```diff
@@ -48,26 +48,33 @@
     </div>
   </div>
 
-  <div class="po-row" *ngIf="isMultipleValues">
-    <po-input
-      class="po-md-4"
-      name="multipleValuesLabel"
-      [(ngModel)]="multipleValuesLabel"
-      p-label="Label"
-      p-help="Series label"
-      p-required
-    >
+  <div class="po-row" *ngIf="!isCircularType">
+    <po-input class="po-md-4" name="label" [(ngModel)]="label" p-label="Label" p-help="Series label" p-required>
     </po-input>
 
     <po-input
-      class="po-md-4"
+      class="po-md-2"
       name="inputDataSeries"
       [(ngModel)]="inputDataSeries"
       p-label="Data"
       p-help="Example: [24, 58, 83, 66, 75, 19]"
       p-required
     >
     </po-input>
+
+    <po-select
```

Comment:
Sample Restore não está limpando esta propreidade e o Series Type das propriedades

Reply:
Deixar aberto o Type para informar os valores, não apenas quando for Line ou Column